src/handle_action.cpp

```diff
@@ -911,7 +917,17 @@ static void smash()
     }
     tripoint_bub_ms smashp = tripoint_bub_ms( *smashp_ );
 
-    if( !g->warn_player_maybe_anger_local_faction( true ) ) {
+    // Little hack: If there's a smashable corpse, it'll always be bashed first. So don't bother warning about
+    // terain smashing unless it's actually possible.
```

Comment:
```suggestion
    // terrain smashing unless it's actually possible.
```

Reply:
Thanks for the catch.

I amended/force pushed it since it's just a typo, to keep the commit history clean. I hope you don't mind.